x/bank/keeper/keeper_test.go

```diff
@@ -664,10 +653,11 @@ func (suite *IntegrationTestSuite) TestMsgMultiSendEvents() {
 		abci.EventAttribute{Key: []byte(sdk.AttributeKeyAmount), Value: []byte(newCoins2.String())},
 	)
 
-	suite.Require().Equal(abci.Event(event1), events[1])
-	suite.Require().Equal(abci.Event(event2), events[2])
-	suite.Require().Equal(abci.Event(event3), events[3])
-	suite.Require().Equal(abci.Event(event4), events[4])
+	// events are shifted due to the funding account events
+	suite.Require().Equal(abci.Event(event1), events[7])
+	suite.Require().Equal(abci.Event(event2), events[8])
+	suite.Require().Equal(abci.Event(event3), events[9])
+	suite.Require().Equal(abci.Event(event4), events[10])
```

Comment:
we had wrong checks before?

Reply:
they were correct, but since now balance is correctly tracked via mint/send we have multiple events when we fund accounts, the event indexes are now shifted correctly to match this

Reply:
It would be helpful if we add a comment explaning what's in `events`

Reply:
IIRC it's commented on the first test relying on this,  I didn't want to copy paste it